index.html

```diff
@@ -591,15 +593,15 @@ <h3>Code Maintenance Policy and Branching</h3>
 </p>
 
 <p>
-Code submitted for projects in this course must reside in the top level directory of the <i>master</i> branch of your repository.  The directory structure provided in the <a href="https://github.com/autorob/kineval-stencil">KinEval</a> stencil must not be modified.  For example, the file "home.html" should appear in the top level directory of your repository.
+Code submitted for projects in this course must reside in the <i>master</i> branch of your repository.  The directory structure provided in the <a href="https://github.com/autorob/kineval-stencil">KinEval</a> stencil must not be modified.  For example, the file "home.html" should appear in the top level directory of your repository.
```

Comment:
I think the "top level directory" instructions conflicts with the "do not modify directory structure" instructions, which I found confusing. Let me know if I'm misunderstanding.